babel.config.js

```diff
@@ -3,59 +3,44 @@ module.exports = {
     [
       '@babel/preset-env',
       {
-        useBuiltIns: 'usage',
-        corejs: 3,
-        targets: { node: '8' },
+        targets: { node: 18 },
       },
     ],
     '@babel/preset-typescript',
   ],
   plugins: [
-    '@babel/plugin-proposal-optional-chaining',
-    '@babel/plugin-proposal-nullish-coalescing-operator',
-    '@babel/plugin-proposal-class-properties',
     [
       '@babel/plugin-transform-runtime',
       {
-        corejs: 3,
+        corejs: "3",
       },
     ],
   ],
   env: {
-    module: {
-      presets: [
-        [
-          '@babel/preset-env',
-          {
-            modules: false,
-            useBuiltIns: 'usage',
-            corejs: 3,
-            targets: { node: '8' },
-          },
-        ],
-        '@babel/preset-typescript',
-      ],
-    },
     browser: {
       presets: [
         [
           '@babel/preset-env',
           {
             modules: false,
             useBuiltIns: 'usage',
-            corejs: 3,
-            targets: { browsers: 'last 2 versions, ie 11, not dead' },
+            corejs: "3.33",
+            targets: { browsers: 'last 2 versions, not dead, > 0.2%' },
```

Comment:
drops ie11 (was dead since last year anyway) adds more coverage:
previous target coverage:
https://browsersl.ist/#q=last+2+versions%2C+ie+11%2C+not+dead

now:
https://browsersl.ist/#q=last+2+versions%2C+not+dead%2C+%3E+0.2%25

this is the recommended by `browserslist`:
https://github.com/browserslist/browserslist#best-practices